### PR TITLE
Splitting traits scripts

### PR DESCRIPTION
**Pull request by TanyaS08**
I've gone and split out the different projects and then added a 4th traits script that pulls them all together and exports the data. I've also added a very quick and dirty README that covers this

### CODE REVIEW

**Comment by TanyaS08** [comment]
Also cleaned .gitignore

File: .gitignore [hunk 1]
```diff
@@ -1,45 +1,19 @@
+# History files
```
Comment (audhalbritter) [inline .gitignore:1]: Lovely 😊 gitignore file!

File: code/species_traits_export.R [hunk 2]
```diff
@@ -0,0 +1,51 @@
+## Read scripts for the three traits datasets ----
+
```
Comment (audhalbritter) [inline code/species_traits_export.R:2]: super looks much cleaner 🧼 now

File: code/species_traits_export.R [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+trait_data_peru <- bind_rows(trait_pftc3,
+                             trait_puna,
+                             trait_pftc5)
+
+## Export data ----
+
+#PFTC3
+trait_pftc3 %>%
+  distinct(project)
+
```
Comment (audhalbritter) [inline code/species_traits_export.R:18]: Good to have for checks but I think these distinct(project) can go 🖐. But I can do this once I go trough the code

File: code/species_traits_export.R [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+trait_pftc5 %>%
+  filter(treatment != "OFF-PLOT")
+write_csv("clean_data/PFTC5_Peru_2020_LeafTraits_clean.csv")
+
+# PFTC3 - Puna Project - PFTC5
+trait_data_peru %>%
+  distinct(project)
+
+#Sean...
+
```
Comment (audhalbritter) [inline code/species_traits_export.R:46]: 🙈

**Review by audhalbritter — APPROVED** [review]

**Pull request merged**